docs/content/how-to/powervs/prerequisites-and-env-guide.md

```diff
@@ -0,0 +1,19 @@
+## Prerequisites
+
+* Set up the [authentication](#setting-up-ibm-cloud-authentication) by following this section
+* `CIS Domain` Need to have existing CIS Domain in `IBM Cloud Internet Services` which can be used as a `BASEDOMAIN` in create part
```

Comment:
good to add link to the IBM Cloud CIS service here

Reply:
added the link!